src/Xamarin.Android.Build.Utilities/Sdks/MonoDroidSdkUnix.cs

```diff
@@ -38,7 +42,7 @@ protected override string FindRuntime ()
 		protected override bool ValidateBin (string binPath)
 		{
 			return !string.IsNullOrWhiteSpace (binPath) &&
-				File.Exists (Path.Combine (binPath, "generator"));
+				File.Exists (Path.Combine (binPath, "generator.exe"));
```

Comment:
The problem with this change is that in a "real" Xamarin.Android install, we'll have:

    $prefix/bin/generator
    $prefix/lib/mandroid/generator.exe

wherein `$prefix/bin/generator` will be a shell script that invokes `$prefix/lib/mandroid/generator.exe`. The [BindingsGenerator](https://github.com/xamarin/xamarin-android/blob/59ec488/src/Xamarin.Android.Build.Tasks/Tasks/Generator.cs#L15) task expects to [execute `generator` on non-Windows platforms](https://github.com/xamarin/xamarin-android/blob/59ec488/src/Xamarin.Android.Build.Tasks/Tasks/Generator.cs#L143-L145), so if `$prefix/bin/generator` doesn't exist then bindings will fail.

Whether this is *desirable* or not is debatable. On the one hand, since `generator.exe` is an assembly then `Xamarin.Android.Build.Tasks.dll` can just reference that assembly and use the types from the `BindingsGenerator` task, no problem.

The flip side is that `generator.exe` does lots of XML parsing and manipulation, which presumably will use some amount of memory. Do we really want this additional memory use in `Xamarin.Android.Build.Tasks.dll`? Or is it better to just keep it in a separate process?

I don't know. (Numbers would be handy...)

In the meantime, `ValidateBin()` should probably check for *either* `generator` *or* `generator.exe`, just to simplify things. Or, `ValidateBin()` should be a no-op. Or we should create the `$prefix/bin/generator` script now.